src/index.js

```diff
@@ -9,5 +9,61 @@ import './css/base.scss';
 
 // An example of how you tell webpack to use an image (also need to link to it in the index.html)
 import './images/turing-logo.png'
+import Player from '../src/Player'
+import Game from '../src/Game'
 
-console.log('This is the JavaScript entry file - your code begins here.');
+let nameInputs = document.querySelector(".player-name-input");
```

Comment:
This looks like a good place that we could replace all the document query selectors with jquery since we have to implement that. I think it would be easy to do before merging it to the master and we can even go through it together in class.

Reply:
Ok sounds good - I've haven't used jQuery at all yet.